dijet/tasks/root.py

```diff
@@ -0,0 +1,104 @@
+# coding: utf-8
+
+import law
+
+from dijet.tasks.base import HistogramsBaseTask
+from columnflow.util import maybe_import
+from columnflow.tasks.framework.base import Requirements
+
+from dijet.tasks.jer import JER
+
+ak = maybe_import("awkward")
+hist = maybe_import("hist")
+np = maybe_import("numpy")
+plt = maybe_import("matplotlib.pyplot")
+mplhep = maybe_import("mplhep")
+it = maybe_import("itertools")
+up = maybe_import("uproot")
```

Comment:
Looks like most of these are unused -> remove. (Btw, `itertools` is a standard library package and can be imported directly without needing `maybe_import`.)
```suggestion
hist = maybe_import("hist")
```